lightning/src/blinded_path/message.rs

```diff
@@ -137,6 +137,26 @@ impl BlindedMessagePath {
 	pub fn blinded_hops(&self) -> &[BlindedHop] {
 		&self.0.blinded_hops
 	}
+
+	pub(crate) fn set_introduction_node_id(&mut self, node_id: PublicKey) {
+		self.0.introduction_node = IntroductionNode::NodeId(node_id);
+	}
```

Comment:
Could we make a `introduction_node_mut` method returning a mutable reference instead? It would make the call site clearer using that instead of both `introduction_node()` and `set_introduction_node()`.

Reply:
> It would make the call site clearer using that instead of both introduction_node() and set_introduction_node().

Not quite following why that is clearer, lmk if there's any way to elaborate.

I think that API may not be supported by the bindings, @TheBlueMatt is that the case?

Reply:
It shouldn't need to be supported by bindings since it only needs to be `pub(crate)`. It's clearer IMO because you directly modify the variable being checked above rather than obfuscating the modification behind a method call.

Reply:
Oh right, duh. Fixed